hack/code/instancetype_testdata_gen/main.go

```diff
@@ -84,16 +87,20 @@ func main() {
 	}
 }
 
-func getDescribeInstanceTypesOutput(ctx context.Context, ec2Client ec2iface.EC2API, instanceTypes []string) string {
-	out, err := ec2Client.DescribeInstanceTypesWithContext(ctx, &ec2.DescribeInstanceTypesInput{
-		InstanceTypes: aws.StringSlice(instanceTypes),
+type EC2API interface {
```

Comment:
IMO: I'm not sure it's super necessary to use an interface here for a script file -- it's not going to be re-used like an exported function might be